lib/downloader.js

```diff
@@ -73,4 +73,17 @@ function downloader(binDir, callback) {
     });
 }
 
-module.exports = downloader;
\ No newline at end of file
+function isWin() {
+    for (let i = 0; i < process.argv.length; i++) {
+        argument = process.argv[i].split('=');
+        if (argument.length == 2 && argument[0] === 'platform' && argument[1] === 'windows') {
+            return true;
+        }
+    }
+    if (process.platform === 'win32' || process.env.NODE_PLATFORM === 'windows') {
```

Comment:
Can you move this condition over the top? I think it's more important the loop. Use the loop as fallback